src/main/java/com/afornalik/ox/ui/UIDrawBoard.java

```diff
@@ -0,0 +1,32 @@
+package com.afornalik.ox.ui;
+
+import com.afornalik.ox.board.Board;
+
+public class UIDrawBoard {
+
+    private final Board board;
+
+    public UIDrawBoard(Board board) {
+        this.board = board;
+    }
+
+    String draw() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("\n  ");
+        for (int i = 1; i < (board.getBound() * 2) + 1; i++) {
+            if (i % 2 == 0) sb.append((i / 2));
```

Comment:
in if() statement there is "2" it's like MAGIC NUMBER. You should create variable BOARD_DEMENTION

Reply:
done